src/test/scala/com/fulcrumgenomics/util/ReadStructureTest.scala

```diff
@@ -187,6 +193,7 @@ class ReadStructureTest extends UnitSpec with OptionValues {
     ReadStructure("5B101T").length shouldBe 2
     ReadStructure("123456789T").length shouldBe 1
     ReadStructure("10T10B10B10S10M").length shouldBe 5
+    ReadStructure("10T10B10B10S10M10C").length shouldBe 6
```

Comment:
<chuckles> _length is defined on a read structure_

Reply:
I know... I don't like it.